todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -120,52 +198,47 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     }
 
     @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        if (requestCode == REQUEST_EDIT_TASK) {
+            // If the task was edited successfully, go back to the list.
+            if (resultCode == Activity.RESULT_OK) {
+                getActivity().finish();
+                return;
+            }
+        }
+        super.onActivityResult(requestCode, resultCode, data);
+    }
+
     public void setLoadingIndicator(boolean active) {
         if (active) {
             mDetailTitle.setText("");
             mDetailDescription.setText(getString(R.string.loading));
         }
     }
 
-    @Override
     public void hideDescription() {
         mDetailDescription.setVisibility(View.GONE);
     }
 
-    @Override
     public void hideTitle() {
         mDetailTitle.setVisibility(View.GONE);
     }
 
-    @Override
+    public void showActive(boolean isActive){
+        mDetailCompleteStatus.setChecked(isActive);
```

Comment:
I believe it should be the opposite, i.e. `mDetailCompleteStatus.setChecked(!isActive);`